src/ecs/rendering/mod.rs

```diff
@@ -1,9 +1,11 @@
 //! `amethyst` rendering ecs module
 
+pub use self::bundle::RenderBundle;
```

Comment:
I think we should decide on whether we want to have a public module or reexported items. Given these bundles all have the same postfix, a conflict is impossible. I just see this inconsistency existed before the PR :worried: